packages/apps/reputation-oracle/server/src/database/database.module.ts

```diff
@@ -22,6 +22,7 @@ import { UserQualificationEntity } from '../modules/qualification/user-qualifica
 import { WebhookIncomingEntity } from '../modules/webhook/webhook-incoming.entity';
 import { WebhookOutgoingEntity } from '../modules/webhook/webhook-outgoing.entity';
 import { EscrowCompletionEntity } from '../modules/escrow-completion/escrow-completion.entity';
+import { EscrowPayoutsBatchEntity } from 'src/modules/escrow-completion/escrow-payouts-batch.entity';
```

Comment:
Please, use relative path.

Reply:
Fixed in https://github.com/humanprotocol/human-protocol/pull/2946/commits/9e7990737a0132b35cf282fda7833902468176a2